io/io/inc/ROOT/RRawFileUnix.hxx

```diff
@@ -45,6 +45,7 @@ public:
    ~RRawFileUnix();
    std::unique_ptr<RRawFile> Clone() const final;
    int GetFeatures() const final;
+   int GetFd() const { return fFileDes; }
```

Comment:
Snuck this in to help with the registered fd test case, I can remove it if you think it merits another PR (or maybe shouldn't be exposed at all!)

Reply:
I think it doesn't hurt.